internal/datastore/spanner/spanner_test.go

```diff
@@ -1,3 +1,4 @@
+// go:build ci
```

Comment:
How is this different from the line below it?

Reply:
they're the same, but 1.17 `go fmt` enforces both lines during the transition from `+build` to `go:build`

https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md